lib/cache/cache_test.go

```diff
@@ -138,6 +138,7 @@ type testPack struct {
 	spiffeFederations       *local.SPIFFEFederationService
 	staticHostUsers         services.StaticHostUser
 	autoUpdateService       services.AutoUpdateService
+	provisioningStatesS     services.ProvisioningStates
```

Comment:
What does the **S** suffix means ? 

can we just name it `provisioningStates`  ?

Reply:
Sure. Looks like there may have been an -`S` suffix convention at one point (see `usersS`, `accessS`, etc) but happy to drop.